roles/jenkins_job_launcher/tasks/main.yml

```diff
@@ -0,0 +1,39 @@
+---
+# tasks file for jenkins-job-launcher
+    
```

Comment:
Could you add a validation of all the required variables before continuing?

For example:
https://github.com/redhatci/ansible-collection-redhatci-ocp/blob/0dcd42d764d1f231833e2a1d04773159413f481a/roles/sideload_kernel/tasks/main.yml#L2-L5